example_data/ogg0m406-kb27-20210720-0305-s91.fits.fz

```diff
@@ -1,4 +1,4 @@
+            ?�[���L��7���~  
4  
+        ?��w�a���+�t���  
?  _        ?Ѝw���@��U��  
E  '�        ?��z��2��	�  
&  4�        ?�"ھԽ�~o��d�  
  B	        ?������\b` ��  
%  O&        ?�ڠ���%H%�D�  
,  \K        ?�rA$Ӑ6�9��Ɋ�T  
*  iw        ?��e��?�,��L  
E  v�        ?ПV�/0�$�,x��  
$  ��        ?���E�/6�0��:y  
*  �
```

Comment:
This is just removing the L1PUBDAT from the example file, so that it gets filled in automatically on ingestion